### PR TITLE
Solutions/UNS: Add diagram

### PR DESCRIPTION
**Pull request by Yndira-E**
## Description

This adds a section with a diagram, following the related issue and as a follow-up to https://github.com/FlowFuse/website/pull/2768.

## Related Issue(s)

[#2755](https://github.com/FlowFuse/website/issues/2755)

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by Hasmin-AC** [comment]
Looks good to me.

Shall we also add a learn more section at the bottom of the page and list our UNS and MQTT blogs? we can then redirect and remove from menu the  [UNS resources page ](https://flowfuse.com/unified-namespace/).

**Comment by Yndira-E** [comment]
@Hasmin-AC done! check it out

**Comment by Hasmin-AC** [comment]
> @Hasmin-AC done! check it out
Thanks!
Looks good @Yndira-E but lets remove the sub headings of the learning resources section.

File: src/solutions/uns.njk [hunk 1]
```diff
@@ -90,60 +90,135 @@ meta:
...
+    <!-- Learn UNS -->
+    <div class="about w-full pt-24">
+        <h3 class="w-full mb-8 text-gray-500 text-center">Learn about the Unified Namespace (UNS)</h3>
+        <div class="grid grid-cols-1 md:gap-12 md:grid-cols-2 pt-4 md:pt-6 items-start m-auto">
+            <!-- Getting Started -->
+            {% set sections = [
+                {
+                    "articles": [
+                        {
+                            "title": "Introduction into the UNS",
```
Comment (Hasmin-AC) [inline src/solutions/uns.njk:162]:
```suggestion
                            "title": "Introduction to UNS",
```

File: src/solutions/uns.njk [hunk 2]
```diff
@@ -90,60 +90,135 @@ meta:
...
         <h4 class="mb-6 w-full text-center">Ready to simplify your UNS journey and unlock the power of your industrial data?</h4>
-        <div class="flex max-sm:flex-col max-md:max-w-sm max-md:mx-auto gap-3 justify-center">
+        <div class="flex max-sm:flex-col max-md:mx-auto gap-3 justify-center">
             <a class="ff-btn ff-btn--primary min-h-[40px]" href="/book-demo/" onclick="capture('cta-contact-us-for-demo', {'position': 'secondary'}, {'page': 'uns'})">CONTACT US FOR DEMO</a>
             <a class="ff-btn ff-btn--primary-outlined min-h-[40px]" href="https://app.flowfuse.com/account/create" onclick="capture('cta-try-it-on-the-cloud', {'position': 'secondary'}, {'page': 'uns'})">TRY IT ON THE CLOUD</a>
         </div>
     </div>
+    <!-- Learn UNS -->
+    <div class="about w-full pt-24">
+        <h3 class="w-full mb-8 text-gray-500 text-center">Learn about the Unified Namespace (UNS)</h3>
```
Comment (Hasmin-AC) [inline src/solutions/uns.njk:155]:
```suggestion
        <h3 class="w-full mb-8 text-gray-500 text-center">Learn about UNS</h3>
```

**Review by Hasmin-AC — APPROVED** [review]

**Pull request merged**